iceoryx_examples/waitset_in_c/ice_c_waitset_timer_driven_execution.c

```diff
@@ -74,6 +74,7 @@ int main()
            "which triggers the cyclicTrigger every second.\n");
 #endif
 
+    //! [initialization and shutdown handling]
     iox_runtime_init("iox-c-waitset-sync");
```

Comment:
Shouldn't it be `iox-c-waitset-timer-driven-execution` to be consistent with the file and executable name?